Module/Rule/Convert/Functions.ps1

```diff
@@ -220,4 +220,51 @@ function Get-HardCodedString
 
     return $hardCodedString
 }
+
+function Get-HardCodedRuleType
```

Comment:
Comment based help is needed for these functions.

Reply:
Done